src/index.react-native.ts

```diff
@@ -1,8 +1,8 @@
 // React Native entry-point
+/* eslint-disable global-require */
 
-import { Buffer } from 'buffer';
-
-global.process = require('process');
+if (typeof global.process === 'undefined') global.process = require('process');
```

Comment:
This needs to be `global.process = require('process');` since `process.version.slice` will be called